frontend/dockerfile/builder/build.go

```diff
@@ -116,6 +116,13 @@ func Build(ctx context.Context, c client.Client) error {
 		return err
 	}
 
+	if _, ok := c.Opts()["cmdline"]; !ok {
+		ref, argv, ok := dockerfile2llb.DetectSyntax(bytes.NewBuffer(dtDockerfile))
```

Comment:
nit: I expect argv to be a vector (slice)

Reply:
Do I just rename the var to `cmdline`?